packages/pilot/src/components/PasswordInput/index.js

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import { contains } from 'ramda'
+// @TODO: replace this for the lib when its be published
+import { validate } from 'P4g4rm3' // eslint-disable-line
+
+import {
+  FormInput,
+  Popover,
+  PopoverContent,
+} from 'former-kit'
+
+import style from './style.css'
+
+const strengthCalculator = ({ score, isValid }) => classnames({
+  [style.strong]: score === 4 && isValid,
+  [style.moderated]: score === 3 && isValid,
+  [style.weak]: score === 2 && isValid,
+  [style.veryWeak]: score === 1 && isValid,
+  [style.invalid]: !isValid,
+})
+
+class PasswordInput extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      popoverIsVisible: false,
+      validations: validate(props.value),
+      value: props.value,
+    }
+
+    this.handleChange = this.handleChange.bind(this)
+    this.handleBlur = this.handleBlur.bind(this)
+    this.handleOpenPopover = this.handleOpenPopover.bind(this)
+  }
+
+  componentDidUpdate (prevProps, prevState) {
+    const { onChange } = this.props
+
+    if ((prevState.value !== this.state.value) && onChange) {
+      onChange(this.state.value)
+    }
+  }
+
+  handleChange ({ target }) {
```

Comment:
Caso tenhas garantia de que sempre vai chegar um evento aqui(ao que parece vai pq está atrelado aos eventos do componente) seria da hora desestruturar o value aqui
```suggestion
   handleChange ({ target: { value } }) {
```